src/ConfigGen/Program.cs

```diff
@@ -102,6 +102,11 @@ public enum SaslMechanism
         ///     SCRAM-SHA-512
         /// </summary>
         ScramSha512
+
+        /// <summary>
+        ///     OAUTHBEARER
+        /// </summary>
+        OauthBearer

```

Comment:
should be `OAuthBearer` to follow C# naming conventions. Also similar changes elsewhere.